Assets/Scripts/GameData/StatData.cs

```diff
@@ -5,11 +5,16 @@ public class StatData {
 
 	public string name;
 	public string description;
+	public float value;
 
```

Comment:
Is value a keyword in C#?  It looks like all three of these words are showing up as pink.